x-pack/plugins/endpoint/public/applications/endpoint/view/managing/details.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useCallback, useMemo } from 'react';
+import {
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutHeader,
+  EuiTitle,
+  EuiDescriptionList,
+  EuiLoadingSpinner,
+  EuiLoadingContent,
+  EuiHorizontalRule,
+} from '@elastic/eui';
+import { useHistory } from 'react-router-dom';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { useManagementListSelector } from './hooks';
+import { urlFromQueryParams } from './url_from_query_params';
+import { uiQueryParams, detailsData } from './../../store/managing/selectors';
+
+const HostDetails = () => {
+  const details = useManagementListSelector(detailsData);
+  const detailsResultsUpper = useMemo(() => {
+    return [
+      {
+        title: 'OS',
+        description: details.host.os.full,
+      },
+      {
+        title: 'Last Seen',
+        description: 'time',
+      },
+      {
+        title: 'Alerts',
+        description: '0',
+      },
+    ];
+  }, [details]);
+
+  const detailsResultsLower = useMemo(() => {
+    return [
+      {
+        title: 'Policy',
+        description: details.endpoint.policy.id,
+      },
+      {
+        title: 'Policy Status',
+        description: 'active',
```

Comment:
You could be almost as cool as @dplumlee if you used `EUIHealth` here https://elastic.github.io/eui/#/display/health

Reply:
i think the status is being mocked right now so i didn't think it was necessary to put it in, but i guess it doesn't hurt to put a mocked health thing right now too.

Reply:
I'm OK with putting it in now as long as we're not inflating this PR too much in both code and testing.  I'd rather we just stay simple for now as we don't have a real Policy status right now.

@parkiino @peluja1012